### DISCUSSION TITLE
DeltaLake vs parquet file

### DISCUSSION THREAD

**Discussion by dridk** [delta-io/delta-rs#1961] [General]
Hi, 

I don't see any real advantage in using delta lake instead of simple parquet files. but I may be wrong.

My ETL process consists of batch extracting of data partitioned by month. So 1 parquet file per month on a local storage.
 
-If I have a problem with a corrupt file or need to update data, I delete the parquet file and regenerate it. 
-Performances are greate with Partition, stats and duckdb.

So I don't need an ACID property or history.

Is there something I've missed?

**Comment by ion-elgreco** [comment]
@dridk delta lake allows you to z-order your data so you can do faster filter operations on your table.
